egs/librispeech/asr/simple_v1/prepare.py

```diff
@@ -44,17 +46,7 @@
             supervisions=manifests['supervisions']).compute_and_store_features(
```

Comment:
There is a warning here with the latest lhotse
```
UserWarning: ProcessPoolExecutor using a "fork" multiprocessing detected. In some circumstances this can crash or hang the program, e.g. when using  data augmentation libsox wrappers like WavAugment or torchaudio. We suggest passing an extra argument to initialize the executor: ProcessPoolExecutor(..., mp_context=multiprocessing.get_context("spawn"))
  warnings.warn('ProcessPoolExecutor using a "fork" multiprocessing detected. '
```
I believe it can be fixed with @pzelasko's PR https://github.com/k2-fsa/snowfall/pull/5/files